java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S2189.html

```diff
@@ -1,33 +1,36 @@
 <h2>Why is this an issue?</h2>
-<p>An infinite loop is one that will never end while the program is running, i.e., you have to kill the program to get out of the loop. Whether it is
-by meeting the loop’s end condition or via a <code>break</code>, every loop should have an end condition.</p>
+<p>An infinite loop will never end while the program runs, meaning you have to kill the program to get out of the loop. Every loop should have an end
+condition, whether by meeting the loop’s termination condition or via a <code>break</code> statement.</p>
 <h3>Noncompliant code example</h3>
-<pre>
+<pre data-diff-id="1" data-diff-type="noncompliant">
```

Comment:
If there are no matching compliant solution, we might as well remove the diff metadata